venues/naacl.org/NAACL/2018/Preprint/python/admin-init.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python
+
+import sys, os
+import argparse
+import openreview
+import config
+
+"""
+
+OPTIONAL SCRIPT ARGUMENTS
+
+    baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+    username - the email address of the logging in user
+    password - the user's password
+
+"""
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+groups = {}
+
+groups[config.AUTHORS] = openreview.Group(config.AUTHORS, **config.group_params)
+
+groups[config.CONF] = client.get_group(config.CONF)
```

Comment:
here you can use the library helper to build the group.

Reply:
originally, I wanted to keep the creation of the main groups separate from the setup of the conference. but I think you're right here, since I'm just going to be running this script anyway